### DISCUSSION TITLE
[@types/node] `readline/promises` as well as `node:readline/promises` missing

### DISCUSSION THREAD

**Discussion by louwers** [DefinitelyTyped/DefinitelyTyped#60466] [Issues with a @types package]
See title.

This does not work:

```ts
import * as readline from 'readline/promises';
```

Neither does this:

```ts
import * as readline from 'node:readline/promises';
```

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
I think this might be because `readline/promises` is still experimental as of now (we're on 18.4.0 at the time of writing), but I'm working on a draft PR for this. I'll link when it's up.

**Comment by nickserv** [comment]
Looks like this has been fixed.
